org.eclipse.jdt.ls.debug/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,8 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.debug.core;bundle-version="3.11.0",
  org.eclipse.jdt.debug,
  org.eclipse.jdt.ls.core,
- org.eclipse.jdt.core
+ org.eclipse.jdt.core,
```

Comment:
Reference to org.eclipse.jdt.core is added back but not used. Please remove if not used.